src/main/resources/view/BudgetListCard.fxml

```diff
@@ -1,22 +1,45 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
+<?import javafx.geometry.Insets?>
 <?import javafx.scene.control.Label?>
+<?import javafx.scene.layout.GridPane?>
 <?import javafx.scene.layout.HBox?>
 <?import javafx.scene.layout.Region?>
 <?import javafx.scene.layout.VBox?>
 
+<?import javafx.scene.control.ProgressBar?>
+
+<?import javafx.scene.layout.ColumnConstraints?>
 <HBox id="cardPane" fx:id="cardPane" xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1">
-    <VBox alignment="CENTER_LEFT">
-        <HBox spacing="5" alignment="CENTER_LEFT">
-            <Label fx:id="id" styleClass="cell_big_label">
-                <minWidth>
-                    <!-- Ensures that the label text is never truncated -->
-                    <Region fx:constant="USE_PREF_SIZE" />
-                </minWidth>
-            </Label>
-            <Label fx:id="description" text="\$description" styleClass="cell_big_label" />
+    <GridPane HBox.hgrow="ALWAYS">
+        <columnConstraints>
+            <ColumnConstraints hgrow="SOMETIMES" minWidth="10" prefWidth="150" />
+        </columnConstraints>
+        <HBox GridPane.columnIndex="0">
+            <VBox alignment="CENTER_LEFT">
+                <HBox alignment="CENTER_LEFT">
+                    <Label fx:id="id" styleClass="cell_big_label">
+                        <minWidth>
+                            <!-- Ensures that the label text is never truncated -->
+                            <Region fx:constant="USE_PREF_SIZE" />
+                        </minWidth>
+                    </Label>
+                    <Label fx:id="description" text="\$description" styleClass="cell_big_label" />
+                </HBox>
+                <Label fx:id="criterion" styleClass="cell_small_label" text="\$criterion" />
+            </VBox>
+            <Region HBox.hgrow="ALWAYS" />
+            <VBox alignment="CENTER_RIGHT">
+                <padding>
+                    <Insets top="5" />
+                </padding>
+                <Label fx:id="amount" text="\$amount" />
+                <ProgressBar fx:id="budgetProgress" progress="0" prefWidth="200" prefHeight="10">
+                    <VBox.margin>
+                        <Insets top="5" />
+                    </VBox.margin>
+                </ProgressBar>
+            </VBox>
```

Comment:
Could consider including lines 22-25 here to ensure the content on the right does not get affected when the description is too long. (We can preclude this in our ug, but it's nice for it to break nicely as well.